components/workspace/Panel.tsx

```diff
@@ -35,23 +36,13 @@ export function WorkspacePanel(props: PropsOf<typeof ActionPanel>) {
   const [showItemCreator, setShowItemCreator] = useState(false)
   const actionSubjects = useActionSubjects()
 
-  const handleSelectAll = () => {
-    const checkboxes = formRef.current?.querySelectorAll<HTMLInputElement>(
-      'input[type="checkbox"][name="workspaceItem"]',
-    )
-    const allSelected = Array.from(checkboxes || []).every(
-      (checkbox) => checkbox.checked,
-    )
-    checkboxes?.forEach((checkbox) => (checkbox.checked = !allSelected))
-  }
-
   const handleRemoveSelected = () => {
     const selectedItems = Array.from(
       formRef.current?.querySelectorAll<HTMLInputElement>(
         'input[type="checkbox"][name="workspaceItem"]:checked',
       ) || [],
     ).map((checkbox) => checkbox.value)
-    removeItemsMutation.mutate(selectedItems)
+    removeItemsMutation.mutate(selectedItems as string[])
```

Comment:
Are the checkbox values always strings, never e.g. `undefined`?

Reply:
yep, these checkboxes always carry the value of the did/uri